src/routes/index.svelte

```diff
@@ -39,6 +40,7 @@
 	<GameListHighlighted />
 	<GameListCarousel gamesData={newestGames} carouselTitle="Nieuwe spellen" />
 	<GameListCarousel gamesData={games} carouselTitle="Favoriete spellen" />
+	<Snackbar snackbarMessage="This is an error message" />
```

Comment:
Tof idee, maar is het nodig om deze in `index.svelte` te hebben staan? Denk dat je het hier kan verwijderen. Was denk ik nog voor testing dingen

Reply:
Ohjah goed dat je het zegt.